app/screens/settings-screen/settings-screen.tsx

```diff
@@ -190,21 +190,41 @@ export const SettingsScreen: React.FC = () => {
 
   const settingsList: SettingRow[] = [
     {
-      category:
-        currentLevel === AccountLevel.NonAuth
-          ? LL.GetStartedScreen.logInCreateAccount()
-          : LL.common.account(),
+      category: LL.SettingsScreen.logInWithPhone(),
+      id: "login-phone",
+      icon: "person-outline",
```

Comment:
Maybe use a phone icon here and an email icon in the next setting, or combine them all and just navigate to the getStarted screen

Reply:
> or combine them all and just navigate to the getStarted screen
I went for this option